api/api.go

```diff
@@ -234,6 +294,32 @@ func (api *ImportAPI) addDimension(w http.ResponseWriter, r *http.Request) {
 	log.Info("dimension added", log.Data{"instance_id": instanceID, "name": dimensionName, "value": dimensionValue})
 }
 
+func (api *ImportAPI) getDimensionValues(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	instanceID := vars["instanceId"]
```

Comment:
query string parameters should be snake cased ? (or was there some talk of having hyphens in URL's?)